ctdcal/process_ctd.py

```diff
@@ -1964,54 +1958,41 @@ def export_ct1(df, ssscc_list):
         time_data = time_data[p_column_names]
         time_data = time_data.round(4)
 
-        s_num = ssscc[-5:-2]
-        c_num = ssscc[-2:]
-        depth = full_depth_df.loc[full_depth_df['STNNBR'] == int(s_num),'DEPTH']
+        depth = full_depth_df.loc[full_depth_df['SSSCC'] == ssscc,'DEPTH'].iloc[0]
         # get cast_details for current SSSCC
         cast_dict = cast_details[cast_details["SSSCC"] == ssscc].to_dict("r")[0]
         b_datetime = datetime.fromtimestamp(cast_dict["bottom_time"], tz=timezone.utc).strftime('%Y%m%d %H%M').split(" ")
-        b_date = b_datetime[0]
-        b_time = b_datetime[1]
         btm_lat = cast_dict["latitude"]
         btm_lon = cast_dict["longitude"]
         btm_alt = cast_dict["altimeter_bottom"]
```

Comment:
I suppose these could also move into ctd_header...

Reply:
We haven't covered yo-yo casts yet as an edge case, but it's not a place for this code review. It's fine here for now.